### ISSUE TITLE
Change UTC Timezone

### ISSUE THREAD

**Issue by realhuseyn** [actions/runner#3309]
Hello. I inspected that your codes print logs to the console with UTC Timezone. It is wrong for general usage. For example, I work with different timezone. When I check the runner logs, it prints with UTC. Could you make changes to that issue?
https://github.com/actions/runner/blob/ce4d7be00f4834b6889a82ea55020bb15fa834bf/src/Runner.Listener/Runner.cs#L375

**Comment by jdeppe-pivotal** [comment]
Agreed. It seems more idiomatic to just use the timezone of the hosting system. Then, if necessary, a user could also just use the `TZ` env variable to adjust to whatever they needed it to be.

**Comment by realhuseyn** [comment]
@jdeppe-pivotal but there is no update
